src/robotto.js

```diff
@@ -82,4 +82,44 @@ robotto.parse = function(robotsFile) {
     return rulesObj;
 };
 
+robotto._stripRoute = function(urlParam) {
+    // Removes domain name and returns only the endPoint
+    return urlParam.replace(/^.*\/\/[^\/]+/, '');
+};
```

Comment:
I think that we do not need this function at all, this should be done using the url module
```javascript
url.parse(urlParam).path;
```